internal/util/apiclient/api_client.go

```diff
@@ -65,6 +65,17 @@ func GetApiClient(profile *config.Profile) (*apiclient.APIClient, error) {
 	clientConfig.AddDefaultHeader("Authorization", fmt.Sprintf("Bearer %s", apiKey))
 	clientConfig.AddDefaultHeader(CLIENT_VERSION_HEADER, internal.Version)
 
+	if c.TelemetryEnabled {
+		clientConfig.AddDefaultHeader(telemetry.ENABLED_HEADER, "true")
+		clientConfig.AddDefaultHeader(telemetry.SESSION_ID_HEADER, internal.SESSION_ID)
+		clientConfig.AddDefaultHeader(telemetry.CLI_ID_HEADER, c.Id)
+		if internal.WorkspaceMode() {
+			clientConfig.AddDefaultHeader(telemetry.SOURCE_HEADER, "cli-project")
```

Comment:
Can we move "cli-project" and such sources to a constant - maybe the telemetry pkg if thats the only place we will use these